build.sh

```diff
@@ -17,6 +17,9 @@ BUILD_DIR=${BUILD_DIR:-${UFS_MODEL_DIR}/build}
 mkdir -p ${BUILD_DIR}
 
 [[ -n "${CCPP_SUITES:-""}" ]] && CMAKE_FLAGS+=" -DCCPP_SUITES=${CCPP_SUITES}"
+CMAKE_FLAGS+=" -DCMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH:-""}"
```

Comment:
I am not sure if this is needed if the GOCART component is set up as a `cmake` target. @DusanJovic-NOAA @aerorahul

Reply:
Actually none of these (including CCPP_SUITES) is needed. User can set/export CMAKE_FLAGS directly. No need to do this translation in this build script.

Reply:
These settings are required since the modulefiles included for the additional GOCART dependencies set both the `CMAKE_PREFIX_PATH` and `CMAKE_MODULE_PATH` environment variables to support CMake `find_package()` commands.
Asking the user to set each of these variables manually constitutes an unnecessary burden.

Reply:
if the modules set `CMAKE_PREFIX_PATH`, that will be picked up from the env. when configuring via cmake and do not need to be passed as command line arguments.

Reply:
@aerorahul I'd be happy to remove these settings but their values do not seem to be picked up from the environment when configuring with CMake. CMake 3.20.2 documentation still says that both `CMAKE_PREFIX_PATH` and `CMAKE_MODULE_PATH` need to be set by the project:
https://cmake.org/cmake/help/v3.20/variable/CMAKE_PREFIX_PATH.html
https://cmake.org/cmake/help/v3.20/variable/CMAKE_MODULE_PATH.html